common/src/cellular_at_core.c

```diff
@@ -300,7 +310,15 @@ CellularATError_t Cellular_ATRemoveTrailingWhiteSpaces( char * pString )
     char * p = NULL;
     CellularATStringValidationResult_t stringValidationResult = CELLULAR_AT_STRING_UNKNOWN;
 
-    atStatus = validateString( pString, &stringValidationResult );
+    if ( pString == NULL )
```

Comment:
What is the scope of these parsing functions? Should they be private rather than public?

Reply:
Please take a look.
Update by [this commmit](https://github.com/FreeRTOS/Lab-Project-FreeRTOS-Cellular-Library/pull/8/commits/39dde55192c91c5dee1cd67cd43413ab47534e1f)